src/puja_poreddiwar/FindEvenNumber.java

```diff
@@ -0,0 +1,45 @@
+package puja_poreddiwar;
+
+public class FindEvenNumber {
+
+	void forloopEvenNo(int startRange, int endRange) {
+		System.out.println("For_loop - first 5 Even numbers are: ");
+		for (int num = startRange; num <= endRange; num++) {
+			if (num % 2 == 0) {
```

Comment:
You need to add a counter variable which check if counter variable value is 5 then break the loop